extensions/2.0/Khronos/KHR_materials_volume/README.md

```diff
@@ -0,0 +1,192 @@
+# KHR_materials_volume
+
+## Contributors
+
+* TODO: Name, affiliation, and contact info for each contributor
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec. Needs to be combined with `KHR_materials_transmission`.
+
+## Overview
+
+This extension adds material parameters to configure the volume beneath the surface of closed meshes, providing effects like refraction, absorption and subsurface scattering. The transparency of the surface is controlled by `KHR_materials_transmission`.
+
+Light hitting an object will first interact with the surface according the BSDF. The BSDF determines how much and in which direction the incident light is reflected and transmitted. Light that is transmitted enters the volume, where it is absorbed or scattered by particles inside the medium. After bouncing several times inside the object, the light will eventually hit the surface again. The BSDF is evaluated a second time, the light may now leave the object or be kept inside it if total internal reflection occurs.
```

Comment:
> light may now leave the object or be kept inside it if total internal reflection occurs <
I think here it should say 'if internal reflection occurs' because there might be reflection event which is not necessarily TIR. TIR is called an event when angle between incident ray direction and surface normal is larger then a critical angle.